src/NumericalAlgorithms/Spectral/Filtering.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+/// \cond
+class Matrix;
+template <size_t>
+class Mesh;
+/// \endcond
+
+namespace Spectral {
+/*!
+ * \ingroup SpectralGroup
+ * \brief Matrices for filtering spectral coefficients.
+ */
+namespace filtering {
```

Comment:
What determines when we prefer a capitalized namespace vs a lower-case namespace? It seems to me we should choose one and try to be consistent.

Reply:
About a year or so ago @kidder (and I think @wthrowe too) and I decided we are consistently inconsistent, don't have a preference for the casing, and don't care enough to change it.